client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -424,3 +425,60 @@ def test_update_autoscaling(self, client, created_endpoints, experiment_run, mod
                 assert metric["parameters"][0]["value"] == "100"
             else:
                 assert metric["parameters"][0]["value"] == "0.7"
+
+    def test_update_from_json_config_with_params(self, client, in_tempdir, created_endpoints, experiment_run, model_for_deployment):
+        yaml = pytest.importorskip("yaml")
+        experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
+        experiment_run.log_requirements(['scikit-learn'])
+
+        path = verta._internal_utils._utils.generate_default_name()
+        endpoint = client.set_endpoint(path)
+        created_endpoints.append(endpoint)
+
+        original_status = endpoint.get_status()
+        original_build_ids = get_build_ids(original_status)
+
+        # Creating config dict:
+        config_dict = {
+            "run_id": experiment_run.id,
+            "strategy": "direct",
```

Comment:
This should _also_ work with canary and its rules, right?

Reply:
No reason to believe otherwise 😬